src/test/java/com/google/maps/StaticMapsApiTest.java

```diff
@@ -0,0 +1,161 @@
+/*
+ * Copyright 2018 Google Inc. All rights reserved.
+ *
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this
+ * file except in compliance with the License. You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+ * ANY KIND, either express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package com.google.maps;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+
+import com.google.maps.StaticMapsRequest.ImageFormat;
+import com.google.maps.StaticMapsRequest.Markers;
+import com.google.maps.StaticMapsRequest.Markers.CustomIconAnchor;
+import com.google.maps.StaticMapsRequest.Markers.MarkersSize;
+import com.google.maps.StaticMapsRequest.Path;
+import com.google.maps.StaticMapsRequest.StaticMapType;
+import com.google.maps.model.LatLng;
+import com.google.maps.model.Size;
+import java.awt.image.BufferedImage;
+import java.io.ByteArrayInputStream;
+import javax.imageio.ImageIO;
```

Comment:
Is this library only pulled in for testing? Do we have it for the regular lib?

Reply:
I only use it in the test library to make sure we can round trip an image. I'm not decoding the image in the main library to keep up throughput.